src/i18n/en.json

```diff
@@ -1827,6 +1827,32 @@
     "expenditure.paymentOverview.totalPayments": "Total payments",
     "expenditure.paymentOverview.paid": "Paid",
     "expenditure.paymentOverview.payableNow": "Payable now",
+    "multiSig.approval.heading": "{threshold} {threshold, plural, one {signature is} other {signatures are}} required to approve the action",
```

Comment:
I would honestly try to pull all of these next to components https://github.com/JoinColony/ColonyFrontEndLivingStandard/blob/master/codeQuality/index.md#i18n